notifications/notifications.yml

```diff
@@ -255,6 +322,31 @@ Resources:
       Dimensions:
         - Name: FunctionName
           Value: !Ref IkeCallbackHandlerLambdaFunction
+  IkeSlashCommandsHandlerErrorAlarm:
```

Comment:
Strange that this would ever be an issue.  But better than silently failing, i suppose.